components/x-gift-article/src/lib/api.js

```diff
@@ -43,9 +43,16 @@ export default class ApiClient {
 		}
 	}
 
-	async getGiftUrl(articleId) {
+	async getGiftUrl(articleId, highlightsToken = undefined) {
 		try {
-			const json = await this.fetchJson('/article/gift-link/' + encodeURIComponent(articleId))
+			let json
+			if (highlightsToken) {
+				json = await this.fetchJson(
+					'/article/gift-link/' + encodeURIComponent(articleId) + `?highlightsToken=${highlightsToken}`
```

Comment:
**Change request:** Here we are using a mixture of string concatenation (`'' + ''`) and template literals (``${}``) please could we keep this consistent and only use a single way of producing the string. I would suggest template literals

 ```suggestion
					`/article/gift-link/${encodeURIComponent(articleId)}?highlightsToken=${highlightsToken}`
```